README.md

```diff
@@ -7,9 +7,26 @@ Labyrinth
 [![Percentage of issues still open](http://isitmaintained.com/badge/open/fossasia/labyrinth.svg)](http://isitmaintained.com/project/fossasia/labyrinth "Percentage of issues still open")
 [![license](https://img.shields.io/github/license/fossasia/labyrinth.svg)](LICENSE)
 
+[![GitHub forks](https://img.shields.io/github/forks/fossasia/labyrinth.svg?style=social&label=Fork)]()
```

Comment:
I removed these three because they are on the github page. But it is ok. @YashKumarVerma also wanted to add them so be it!